platform/viewer/cypress/integration/common/OHIFStudyViewer.spec.js

```diff
@@ -267,7 +267,7 @@ describe('OHIF Study Viewer Page', function() {
   });
 
   it('opens About modal and verify the displayed information', function() {
-    cy.get('[data-cy="options-menu"]').click();
+    cy.get('[data-cy="options-menu"]').click({ multiple: true });
```

Comment:
I think this means we click once per item returned. Is that really what we want? We probably on the "first" item to be clicked.

Reply:
This was fixed with the latest changes. thx.